Makefile

```diff
@@ -3,14 +3,10 @@ GO_DIRS = $(shell ls -d */ | \grep -v vendor)
 GO_PKGS = $(shell go list ./... | \grep -v vendor)
 GO_SRCS = $(shell find . -type f -name '*.go')
 
-# assets removes the existing bindata.go and regenerates it.
```

Comment:
Sorry, I think you misunderstand what I said. We need assets generation but I don't want to format the generated file `bindata.go`.

Reply:
I am sorry but I can't do that, if I am to add the generation, it will be properly formatted. However, as it was not there before and it is not there now so I'd say we can move ahead. Feel free to add it in a later time if you'd like. I am mostly concerned about the race condition.

Reply:
Sorry about that because of case change, I missed it. Makefile reverted.

Reply:
However, note that 1) the case is wrong, 2) make vet fails, 3) make lint fails

Reply:
Unit tests failed to pass. Have you run unit tests on your local machine? What is the result?

Reply:
Work fine. However, as the build was failing before too.

Reply:
Let's fix this issue before we merge since this is the first time I saw the tests are failing. The reason it's not passing before is that we didn't finish the implementation about travis. However, you finished the rest part of it and it should work. Something goes wrong. I will check in your commits in my local box to see what is going on.

Reply:
The tests are failing on master... just run them in a loop for a few seconds and you will see them fail.

On master:

```
while go test -count=1 ./xray; do; done
```

Or make them fail faster:

```
while go test -race -count=1 ./xray; do; done
```